Flux end-to-end test failing with "field type conflict"

The Flux end-to-end test `chronograf/aggregate_window_median_test.flux` is failing post-split with Flux upgraded to v0.83.1, with this error:
```
   --- FAIL: TestFluxEndToEnd/testing/chronograf/aggregate_window_median (0.14s)
            end_to_end_test.go:226: error calling function "inspect" @0:0-0:0: error calling function "case" @testing.flux|20:10-20:16: error calling function "loadStorage" @aggregate_window_median_test.flux|105:11-105:43: error calling function "chain" @9:9-12:3: field type conflict
```
This error message is not present in the `flux` repo and seems to be coming from storage.

The test will be skipped in `influxdb/query/stdlib/testing/testing.go`.

--- a/go.mod
+++ b/go.mod
@@ -46,7 +46,7 @@ require (
 	github.com/hashicorp/vault/api v1.0.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/influxdata/cron v0.0.0-20191203200038-ded12750aac6
-	github.com/influxdata/flux v0.83.1
+	github.com/influxdata/flux v0.83.2
 	github.com/influxdata/httprouter v1.3.1-0.20191122104820-ee83e2772f69
 	github.com/influxdata/influxql v0.0.0-20180925231337-1cbfca8e56b6
 	github.com/influxdata/pkg-config v0.2.3

--- a/query/promql/internal/promqltests/go.mod
+++ b/query/promql/internal/promqltests/go.mod
@@ -10,7 +10,7 @@ require (
 	github.com/go-kit/kit v0.10.0 // indirect
 	github.com/google/go-cmp v0.5.0
 	github.com/hashicorp/go-rootcerts v1.0.2 // indirect
-	github.com/influxdata/flux v0.83.1
+	github.com/influxdata/flux v0.83.2
 	github.com/influxdata/influxdb/v2 v2.0.0-00010101000000-000000000000
 	github.com/influxdata/influxql v1.0.1 // indirect
 	github.com/influxdata/promql/v2 v2.12.0

--- a/query/stdlib/testing/testing.go
+++ b/query/stdlib/testing/testing.go
@@ -141,9 +141,8 @@ var FluxEndToEndSkipList = map[string]map[string]string{
 		"join": "unbounded test",
 	},
 	"testing/chronograf": {
-		"buckets":                 "unbounded test",
-		"aggregate_window_count":  "flakey test: https://github.com/influxdata/influxdb/issues/18463",
-		"aggregate_window_median": "failing with \"field type conflict\": https://github.com/influxdata/influxdb/issues/19565",
+		"buckets":                "unbounded test",
+		"aggregate_window_count": "flakey test: https://github.com/influxdata/influxdb/issues/18463",
 	},
 	"testing/kapacitor": {
 		"fill_default": "unknown field type for f1",

